crates/bindings-csharp/Runtime/RawBindings.cs

```diff
@@ -63,28 +62,36 @@ public class StdbException : Exception
 
     public class NoSuchTableException : StdbException
     {
-        internal NoSuchTableException() : base(ERRNO_NO_SUCH_TABLE) { }
+        internal NoSuchTableException()
```

Comment:
How come this formatting change ended up in this PR? Did the autoformatter in use change or something?

Reply:
I think @coolreader18 just didn't have autoformatted configured in their editor, but I do, so it formatted code on save.